### PR TITLE
EASY-2789: Add generic key-value pair for file-metadata from fedora

### PR DESCRIPTION
**Pull request by lindareijnhoudt**
accompanies EASY-2789

#### When applied it will
* Add generic key-value pair for file-metadata from fedora
* 
* 

#### related pull requests on github
repo                       | PR
-------------------------- | -----------------
easy-                      | [PR#](PRlink)

### CODE REVIEW

File: lib/src/main/resources/bag/metadata/afm/afm.xsd [hunk 1]
```diff
@@ -75,7 +78,11 @@
             <xs:documentation xml:lang="dut">technische bestandsspecificatie</xs:documentation>
         </xs:annotation>
     </xs:element>
-
+    
+    <xs:element name="file_name" substitutionGroup="dcterms:title"/>
```
Comment (jo-pol) [inline lib/src/main/resources/bag/metadata/afm/afm.xsd:82]: So I should change https://github.com/DANS-KNAW/easy-fedora-to-bag/blob/1d6812896df165ef2a5ae62d9912114d3b75eef6/src/main/scala/nl/knaw/dans/easy/fedoratobag/FileItem.scala#L88 ?

File: lib/src/main/resources/bag/metadata/afm/afm.xsd [hunk 2]
```diff
@@ -75,7 +78,11 @@
...
+    
+    <xs:element name="description" substitutionGroup="dcterms:description"/>
```
Comment (jo-pol) [inline lib/src/main/resources/bag/metadata/afm/afm.xsd:84]: Child or sibling of `addmd`?

File: lib/src/main/resources/bag/metadata/afm/afm.xsd (hunk 1 above)
Comment (lindareijnhoudt) [reply lib/src/main/resources/bag/metadata/afm/afm.xsd:82]: no, the way you did it is better

File: lib/src/main/resources/bag/metadata/afm/afm.xsd (hunk 2 above)
Comment (lindareijnhoudt) [reply lib/src/main/resources/bag/metadata/afm/afm.xsd:84]: i would say same as the other elements here? maybe i'm not following

**Pull request merged**